app/serializers/webfinger_serializer.rb

```diff
@@ -13,7 +13,7 @@ def aliases
     if object.instance_actor?
       [instance_actor_url]
     else
-      [short_account_url(object), account_url(object)]
+      object.to_webfinger_aliases + [short_account_url(object), account_url(object)]
```

Comment:
Side-note: Are the URLs here used for anything at all? As far as my memory goes, I added them all the way back when first implementing webfinger because that's what GNU social was doing. But if no software actually uses these values, might be worth removing to have cleaner code here.

Reply:
I have never checked, but I don't think any fediverse software actually uses these values.